packages/beacon-node/src/chain/archiver/interface.ts

```diff
@@ -0,0 +1,46 @@
+import {CheckpointWithHex} from "@lodestar/fork-choice";
+import {Metrics} from "../../metrics/metrics.js";
+import {RootHex} from "@lodestar/types";
```

Comment:
the new linter does not enforce import order anymore?

Reply:
There is a caveat related to import order, not yet enabled.